style.css

```diff
@@ -0,0 +1,96 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-image: url(./imges/background\ image.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
```

Comment:
- [ ] At the moment, the background image you added is not displaying properly according to the given template; the reason is because you placed it under a `*`. `*` is a universal selector, its selects all HTML elements on the page. ℹ️ If you add the suggested code below to the `body` (line 12) element, I believe the background-image will be perfectly aligned.

```suggestion
  background-image: url(./imges/background\ image.png);
  background-repeat: no-repeat;
  background-position: center;
```

## Your design
<img width="300" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/227555658-3e200fda-6365-4f2e-ac34-5b8870a0dc18.png">